src/parsers/power_models_data.jl

```diff
@@ -295,35 +297,37 @@ function read_bus!(sys::System, data::Dict; kwargs...)
         add_component!(sys, bus; skip_validation = SKIP_PM_VALIDATION)
     end
 
-    # get Inter-area Transfers as AreaInterchange
-    for (k, d) in data["interarea_transfer"]
-        area_from_name = _get_name_area(d["area_from"])
-        area_to_name = _get_name_area(d["area_to"])
+    if data["source_type"] == "pti" && haskey(data, "interarea_transfer")
+        # get Inter-area Transfers as AreaInterchange
+        for (k, d) in data["interarea_transfer"]
+            area_from_name = _get_name_area(d["area_from"])
+            area_to_name = _get_name_area(d["area_to"])
 
-        from_area = get_component(Area, sys, area_from_name)
-        to_area = get_component(Area, sys, area_to_name)
+            from_area = get_component(Area, sys, area_from_name)
+            to_area = get_component(Area, sys, area_to_name)
 
-        name = "$(area_from_name)_$(area_to_name)"
-        available = true
-        active_power_flow = d["power_transfer"]
-        flow_limits = (from_to = Inf, to_from = Inf)
+            name = "$(area_from_name)_$(area_to_name)"
+            available = true
+            active_power_flow = d["power_transfer"]
+            flow_limits = (from_to = Inf, to_from = Inf)
```

Comment:
I am not sure we can serielize/deserialize `Inf` so we need to use the `INFINITE_BOUND` constant 
```suggestion
            flow_limits = (from_to = INFINITE_BOUND, to_from = INFINITE_BOUND)
```